subsys/bluetooth/mesh/shell/cfg_cli.c

```diff
@@ -33,6 +33,14 @@ static int cmd_reset(const struct shell *sh, size_t argc, char *argv[])
 		return 0;
 	}
 
+	if (IS_ENABLED(CONFIG_BT_MESH_CDB)) {
```

Comment:
Can `cmd_cdb_node_del` be used to remove a node from CDB?

Reply:
Well it could, but then I guess we must make it accessable from shell.c->cfg_cli.c. Isn't that a little bit akward?

Reply:
Or did you mean that the user can do that in the shell menu? In that case yes, but I found it a little painful to manually delete it when I was continiously provisioning and reseting a node. Is there many usecases where you would want to keep the node CDB entry after reset?

Reply:
The same problem when resetting provisioner with `mesh reset`, when you have to call `mesh cdb reset` manually.

I looked at the code again. We add entries to cdb when adding new app/net keys, but neither updating then or deleting. Maybe we should fix all the commands together?

Reply:
Ok, so fix appkey/netkey `Update`, `Delete` and `mesh reset` then?

Reply:
Not sure about `mesh reset`, others I guess should be updated?

Reply:
Correction*: Not `Delete`

Reply:
The more I think about it the complexity of updating or deleting keys for CDB seems tricky. I'll add the cdb clear to the local reset for now since it is a no-brainer. We should think a little about delete and update and maybe do it as a separate task.